### DISCUSSION TITLE
Problems seting profile permissions

### DISCUSSION THREAD

**Discussion by alexynadamas** [YetiForceCompany/YetiForceCRM#14700] [How to] [answered]
Hi

When I select all permissions (create, read, update and delete) to a module for a profile and then I save It, the permissions comeback to disable form.

https://user-images.githubusercontent.com/60443759/119105149-53579a00-ba1d-11eb-9ae5-a96e97053266.mp4

Probably other permission is doing that.

Thanks.

**Comment by davide-alghi** [comment, marked as answer]
hi @alexynadamas,
that is an old bug. fixed in YF 5.3

**Reply by MexasPT** [reply]
I use version 6.1 with ServicePack 8 and this also happens.
There are some modules that do not save the selected changes when saving.

Does anyone know where to manually change this situation in the database?

**Reply by alexynadamas** [reply]
I fixed it creating again a new profile with the same permissions

**Comment by garryatbrighton** [comment]
same here... duplicating profile is not a solution. What causing that?

**Comment by mariuszkrzaczkowski** [comment]
I saw this effect when:
- wrong server configuration (most often)
- the profile on which the admin is logged in has been disabled for action permissions, so he cannot save them in other profiles (error inherited from vtiger)
- there are no entries in the database, e.g. actions
